scripts/stage1.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This script is run at package build time to create the base recovery
```

Comment:
My understanding was that the bootstrap VM was constant for a given LTS (for the relevant files; we might get new packages, but the kernel version would be staying the same). If that is not the case, then we need to come up with a different design if we want repeatable builds.

Reply:
@pcd1193182 I'd not rely on files from the build server since we have rebuilt it a few times before and may configure it in such a way that is different from what we'd want in the recovery environment.

For configuration files copied from `/etc/` we could perhaps keep static copies in this repository. This would also allow us to customize them as needed.

For things like libraries we could perhaps `apt-get download`/extract the packages and then copy them from there? This way we'd use whatever is provided by the linux-package-mirror like we do for the rest of our appliance dependencies.